Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -28,6 +29,38 @@
     'Critical': 4
 }
 
+RESPONSE_ERROR_EXPLANATIONS = {
```

Comment:
since this dictionary is only used in one place (and is unclear when just reading the file), either move it to the same function where it's used (preferred imo), or just above the function

Reply:
Moved it just above the function where it is used in commit 2d3ae24d9b52ec8d660e15df52420ed82c545827

Perhaps we should remove this dictionary all together and just display the error "as is" from the response?

Reply:
I have resolved this conversation based on our discussion in Slack